code/_helpers/lists.dm

```diff
@@ -180,6 +180,11 @@ Checks if a list has the same entries and values as an element of big.
 		return picked
 	return null
 
+/proc/popleft(list/L)
```

Comment:
Documentation here - other ones aren't as bad because they're self-explanatory in the name, but this one isn't

Reply:
Added